test/quantization/core/experimental/test_nonuniform_observer.py

```diff
@@ -2,8 +2,24 @@
 
 from torch.ao.quantization.experimental.observer import APoTObserver
 import unittest
+import matplotlib.pyplot as plt
+import torch
 
 class TestNonUniformObserver(unittest.TestCase):
+    def quant_levels_visualization(self, obs, obs_result, filename):
```

Comment:
nit: this would normally be a private method since its not an actual test, prepend with '_'